### PR TITLE
Fixed talon warning messages

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
* Warning about `-` in the help context/list because of the filename `cursorless-snippets` 
* Warning about missing modifiers capture because of a circular dependency

## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)

### CODE REVIEW

File: cursorless-talon/src/modifiers/modifiers.py [hunk 1]
```diff
@@ -30,23 +30,39 @@ def cursorless_simple_modifier(m) -> dict[str, str]:
...
+
 
 @mod.capture(rule="|".join(modifiers))
 def cursorless_modifier(m) -> str:
     """Cursorless modifier"""
     return m[0]
 
 
+@mod.capture(rule="|".join(head_tail_swallowed_modifiers))
+def cursorless_head_tail_swallowed_modifier(m) -> str:
```
Comment (pokey) [inline cursorless-talon/src/modifiers/modifiers.py:61]: This needs to be in this file to avoid circular import with `head_tail.py`

**Review by pokey — APPROVED** [review]
I made some minor tweaks so have a look before you merge but I'm happy now

File: cursorless-talon/src/modifiers/modifiers.py (hunk 1 above)
Comment (AndreasArvidsson) [reply cursorless-talon/src/modifiers/modifiers.py:61]: It's not the circular imports of the files but the circular dependencies of the captures I think is the problem. But we could definitely have it in here if you prefer.

Comment (pokey) [reply cursorless-talon/src/modifiers/modifiers.py:61]: Idk I was getting the following before moving this:

```
2022-07-04 10:46:04 ERROR user.cursorless-talon.src.modifiers.head_tail (/Users/pokey/.talon/user/cursorless-talon/src/modifiers/head_tail.py) import failed
   17:                       lib/python3.9/threading.py:930* # loader thread
   16:                       lib/python3.9/threading.py:973* 
   15:                       lib/python3.9/threading.py:910* 
   14:                          app/resources/loader.py:805| 
   13:                          app/resources/loader.py:741| 
   12:                          app/resources/loader.py:561| 
   11:                          app/resources/loader.py:597| 
   10:                          app/resources/loader.py:649| 
    9:                          app/resources/loader.py:423| # [stack splice]
    8:              lib/python3.9/importlib/__init__.py:127| 
    7:                          app/resources/loader.py:246| 
    6:                          app/resources/loader.py:241| 
    5: user/cursorless-talon/src/modifiers/head_tail.py:3  | from .modifiers import head_tail_swall..
    4:                          app/resources/loader.py:357| 
    3:                          app/resources/loader.py:246| 
    2:                          app/resources/loader.py:241| 
    1: user/cursorless-talon/src/modifiers/modifiers.py:4  | from .head_tail import head_tail_modifiers
ImportError: cannot import name 'head_tail_modifiers' from partially initialized module 'user.cursorless-talon.src.modifiers.head_tail' (most likely due to a circular import) (/Users/pokey/.talon/user/cursorless-talon/src/modifiers/head_tail.py)
```

**Pull request merged**